examples/classification/README.md

```diff
@@ -1,6 +1,8 @@
 # Classification
 
-## Performance
+## ImageNet
+
+### Weight conversion
 
 Single crop error rate.
```

Comment:
For the consistency with `Trained model`, it would be nice to add something like `~ of the models converted from Caffe model`.